README.md

```diff
@@ -70,11 +70,15 @@ Check the [**user**](https://parallaxsecond.github.io/parsec-book/parsec_users.h
 Come and talk to us in our[**Slack channel**](https://app.slack.com/client/T0JK1PCN6/CPMQ9D4H1)!
 [**Here**](http://dockr.ly/slack) is how to join the workspace.
 
-Also join the [**biweekly meeting**](https://calendar.google.com/calendar?cid=ZG9ja2VyLmNvbV9xcHAzbzl2aXBhbmE0NGllcmV1MjlvcHZkNEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t)
+Also join the [**biweekly meeting**](https://calendar.google.com/event?action=TEMPLATE&tmeid=MDVzbW9nYm8ydDZsZGdwNjN1NmF2c3V1NmZfMjAyMDA0MjhUMTUzMDAwWiBvdHQxOXFhZjN0NWlkOG81anE4dWxpaDFhY0Bn&tmsrc=ott19qaf3t5id8o5jq8ulih1ac%40group.calendar.google.com&scp=ALL)
```

Comment:
Since we have the calendar view below, maybe this can be the link to the Zoom invitation directly? That way it is easy to just click on the link to join the Zoom call and we let people set their own recurring meeting on their calendar application of choice 😃